backend/src/main/scala/sbt/internal/inc/test.worksheet.sc

```diff
@@ -0,0 +1,5 @@
+val a: Vector[String] = Vector()
+a match {
```

Comment:
@dotta When you add the tests, could you remove this file too please? :smile:

PS. I rebased the whole PR so that when merged it goes directly to master.

Reply:
Sounds good! I'm OOO atm and will be back on Thu. Hopefully, I can have it all done by the end of the week!